firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/internal/LogWrapper.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
```

Comment:
Rather than duplicating this class, I'd suggest making the Logger class public. Crashlytics and Fireperf both have public logger classes